testsuites/syncgateway/functional/tests/test_db_online_offline_resync.py

```diff
@@ -299,6 +309,23 @@ def test_bucket_online_offline_resync_with_online(params_from_base_test_setup, s
     except Exception as e:
         log_info("Catch resync exception: {}".format(e))
 
+    if sync_gateway_version < "3.0.0":
+        resync_result = async_resync_result.get()
+        log_info("resync_changes {}".format(resync_result))
+        log_info("expecting num_changes  == num_docs {} * num_users {}".format(num_docs, num_users))
+        assert resync_result['payload']['changes'] == num_docs * num_users
+        assert resync_result['status_code'] == 200
+    else:
+        retries = 0
+        resync_result = admin.db_get_resync_status(db="db")
+        while resync_result != "stopped" and retries < 5:
+            resync_result = admin.db_get_resync_status(db="db")
+            print("resync_result is ", resync_result)
```

Comment:
We can remove the print statement safely

Reply:
done